### PR TITLE
Fix 'undefined: logging.LevelLogger'

### PR DESCRIPTION
**Pull request by ahadas**
pkg/controller/plan/adapter/ocp/validator.go:24:23: undefined: logging.LevelLogger

### CODE REVIEW

**Comment by ahadas** [comment]
@bennyz do you have a better idea? see [this log](https://github.com/kubev2v/forklift/actions/runs/5870529245/job/15917721025)

**Comment by bennyz** [comment]
I didn't understand why this didn't fail, but then noticed it's the stable branch...

File: pkg/controller/plan/adapter/ocp/validator.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ type Validator struct {
 	plan         *api.Plan
 	inventory    web.Client
 	sourceClient k8sclient.Client
-	log          logging.LevelLogger
```
Comment (bennyz) [inline pkg/controller/plan/adapter/ocp/validator.go:24]: can't we replace it with `logr.Logger`? then it should be the only change

Comment (ahadas) [reply pkg/controller/plan/adapter/ocp/validator.go:24]: you're right, changed

**Review by bennyz — APPROVED** [review]

**Pull request merged**